docs/products/clickhouse/howto/integrate-kafka.md

```diff
@@ -67,10 +68,9 @@ configuration, see the section below.
 
 ## Update Apache Kafka integration settings
 
-Configure the topic and data format options for the
-integration. This will create a virtual table in Aiven for ClickHouse
-that can receive and send messages from multiple topics. You can have as
-many of such tables as you need.
+Configure the topic and data format options for the integration. This will create a virtual
```

Comment:
One small suggestion: it might help to clarify where or how to configure these options. For example, you could say “configure the topic and data format options in the integration settings” to make the connection to the previous step more explicit.